sensitivity-api/src/main/java/com/powsybl/sensitivity/SensitivityComputation.java

```diff
@@ -1,53 +1,266 @@
-     * Run an asynchronous single sensitivity computation job using given parameters and input provider
+     * Get a runner for default sensitivity computation implementation.
      *
-     * @param factorsProvider sensitivity factors provider for the computation
-     * @param workingStateId id of the network base state for the computation
-     * @param sensiParameters sensitivity computation parameters
-     * @return the sensitivity computation results on the given situation
+     * @throws PowsyblException in case we cannot find a default implementation
+     * @return a runner for default sensitivity computation implementation
      */
-    CompletableFuture<SensitivityComputationResults> run(SensitivityFactorsProvider factorsProvider, String workingStateId, SensitivityComputationParameters sensiParameters);
+    public static SensitivityComputationRunner find() {
+        return find(null);
+    }
 
     /**
-     * Run an asynchronous sensitivity analysis job using given parameters and input provider
+     * A variant of {@link SensitivityComputation#find(String)} intended to be used for unit testing that allow passing
+     * an explicit provider list instead of relying on service loader and an explicit {@link PlatformConfig}
+     * instead of global one.
      *
-     * @param factorsProvider sensitivity factors provider for the computation
-     * @param contingenciesProvider contingencies provider for the computation
-     * @param workingStateId id of the network base state for the computation
-     * @param sensiParameters sensitivity computation parameters
-     * @return the sensitivity computation results in N and N-1
+     * @param name name of the sensitivity computation implementation, null if we want to use default one
+     * @param providers sensitivity computation provider list
+     * @param platformConfig platform config to look for default sensitivity computation implementation name
+     * @return a runner for sensitivity computation implementation named {@code name}
      */
-    default CompletableFuture<SensitivityComputationResults> run(SensitivityFactorsProvider factorsProvider, ContingenciesProvider contingenciesProvider, String workingStateId, SensitivityComputationParameters sensiParameters) {
-        throw new UnsupportedOperationException("Not implemented");
+    static SensitivityComputationRunner find(String name, List<SensitivityComputationProvider> providers, PlatformConfig platformConfig) {
+        Objects.requireNonNull(providers);
+        Objects.requireNonNull(platformConfig);
+
+        if (providers.isEmpty()) {
+            throw new PowsyblException("No sensitivity computation provider found");
+        }
+
+        // if no sensitivity computation implementation name is provided through the API we look for information
+        // in platform configuration
+        String sensitivityComputationName = name != null ? name : platformConfig.getOptionalModuleConfig("sensitivity-computation")
```

Comment:
It seems to me this config module is new, what parameters would be common to any sensitivity analysis provider?